docs/network-iot/run-an-lns/configure-chirpstack.mdx

```diff
@@ -91,6 +92,70 @@ configuration example and documentation.
 
 Make sure ports 1700 and 1701 are both open on the server.
 
+
+### Example `chirpstack.toml` with Helium Additions.
```

Comment:
```suggestion
## Example `chirpstack.toml` with Helium Additions.
```
make this a l2 since it's not a subsection of 'Support Additional..." above

Reply:
sorry i missed that, i'll edit and re-push shortly